secure.go

```diff
@@ -232,3 +255,27 @@ func (s *Secure) Process(w http.ResponseWriter, r *http.Request) error {
 
 	return nil
 }
+
+// CSPNonce returns the nonce value associated with the present request. If no nonce has been generated it returns an empty string.
+func CSPNonce(c context.Context) string {
+	if val, ok := c.Value(nonceKey).(string); ok {
+		return val
+	}
+
+	return ""
+}
+
+type key int
+
+const nonceKey key = iota
+
+func randNonce() string {
```

Comment:
Instead of `randNonce`, lets be more specific in case another nonce is added in the future. Maybe the same for `nonceKey`.